pkg/kudoctl/util/kudo/install.go

```diff
@@ -145,3 +151,20 @@ func versionExists(version string, versions []string) bool {
 
 	return false
 }
+
+func render(name, manifest string, resources *packages.Resources, instanceName, namespace string, parameters map[string]string) (string, error) {
+	configs := make(map[string]interface{})
+	configs["OperatorName"] = resources.Operator.Name
+	configs["Name"] = instanceName
+	configs["Namespace"] = namespace
+	configs["Params"] = parameters
+	configs["AppVersion"] = resources.OperatorVersion.Spec.AppVersion
+	configs["OperatorVersion"] = resources.OperatorVersion.Spec.Version
```

Comment:
I wonder if we could somehow put these and the assignments from `render.go:15-25` together. Or maybe at least define constants for the strings here. 

Reason is that when at some point we add some more variables for templating, it might be easy to miss one of the places.

Reply:
we actually have in 3 parts of code I discovered... makes sense to me